network/edge-services/how-to/configure-custom-domain.mdx

```diff
@@ -0,0 +1,123 @@
+---
+meta:
+  title: How to configure a custom domain for Edge Services
+  description: Learn how to configure an Edge Services endpoint with a custom subdomain. Access your Object Storage bucket or Load Balancer origin via your own domain name instead of the standardized endpoint.
+content:
+  h1: How to configure a custom domain for Edge Services
+  paragraph: Learn how to configure an Edge Services endpoint with a custom subdomain. Access your Object Storage bucket or Load Balancer origin via your own domain name instead of the standardized endpoint.
+dates:
+  validation: 2024-07-24
+  posted: 2024-07-24
+tags: object-storage edge-services cdn network domain ssl tls https
+categories:
+  - network
+---
+
+If you already own a domain, you customize an Edge Services pipeline endpoint with a subdomain of your choice. This means you can access your Object Storage bucket or Load Balancer origin through Edge Services via your own subdomain rather than the standardized Edge Services endpoint.
```

Comment:
```suggestion
If you already own a domain, you can customize an Edge Services pipeline endpoint with a subdomain of your choice. This means you can access your Object Storage bucket or Load Balancer origin through Edge Services via your own subdomain rather than the standardized Edge Services endpoint.
```